src/cic/rscript.clj

```diff
@@ -4,17 +4,18 @@
             [clj-time.format :as f]
             [clojure.java.shell :as shell]
             [clojure.set :as cs]
-            [clojure.string :as str]))
+            [clojure.string :as str]
+            [taoensso.timbre :as log]))
 
 (defn exec
   [script-path & args]
-  (println (format "Executing %s %s" script-path (str/join " " args)))
+  (log/info (format "Executing %s %s" script-path (str/join " " args)))
```

Comment:
Is it worth putting in a file exists check here and then throwing if it doesn't?

Reply:
I guess we might be grateful in future to know whether the problem is locating the file or executing the file since they're such different failure modes 👍